mod-orders-storage/schemas/prefix.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Purchase order number prefix",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID identifying this prefix",
+      "$ref": "../../common/schemas/uuid.json"
+    },
+    "prefix": {
+      "description": "Prefix",
+      "type": "string"
+    }
```

Comment:
we need a field for "description" too...

Reply:
The current prefix json (in mod-configuration) has two fields:  name and description.  Does it make sense to mirror that here to ease the transition on the client side?